test/storage/online_file_source.test.cpp

```diff
@@ -13,6 +13,13 @@
 
 using namespace mbgl;
 
+#ifdef WIN32
+// Windows doesn't fail immediately like other OS
```

Comment:
Can that be fixed?

Reply:
I don't know if there is a better approach. Unix-like systems fail immediately, and Windows fails after waiting 2 seconds. How to deal with it? A separate file, maybe? A constant defined elsewhere? I'm open to suggestions.

Reply:
It's fine this way.